__tests__/games.test.js

```diff
@@ -98,4 +92,90 @@ afterAll(() => {
           
       });
       
+  });
+
+  describe('PATCH /api/reviews/review_id', () => {
+      test('202: responds with a review with it\'s votes counter altered by the correct amount', () => {
```

Comment:
don't forget to update the test text to reflect the status code you're using now